### DISCUSSION TITLE
Lighthouse SEO enhancement

### DISCUSSION THREAD

**Discussion by kulbhushanchand** [mmistakes/minimal-mistakes#3038] [Q&A]
The google lighthouse points out an enhancement in SEO for the page https://mmistakes.github.io/minimal-mistakes/post/post-future-date/

**Any suggestions?**

![image](https://user-images.githubusercontent.com/17742733/124259574-4960a500-db4c-11eb-959d-7d42f92cd028.png)

**Comment by iBug** [comment]
Those are the "share this article" buttons and it makes no sense to "source" them.

**Reply by kulbhushanchand** [reply]
In that case shouldn't we use `rel=nofollow` since we don't want the links to be followed or use something like this [SO answer].(https://stackoverflow.com/questions/64640807/share-facebook-or-line-with-onclick-function-but-get-seo-problem-links-are-not)
